code/modules/clothing/clothing.dm

```diff
@@ -792,3 +792,10 @@ BLIND     // can't see anything
 		deconstruct(FALSE)
 	else
 		..()
+
+/obj/item/clothing/neck
+	name = "necklace"
+	icon = 'icons/obj/clothing/neck.dmi'
+	body_parts_covered = UPPER_TORSO
+	slot_flags = SLOT_NECK
+	strip_delay = 40
```

Comment:
```suggestion
	strip_delay = 4 SECONDS
``` SECONDS is defined as *10 so it converts miliseconds to seconds, making it easier to read.